vdpwi/__main__.py

```diff
@@ -26,7 +26,7 @@ def evaluate_dataset(split_name, dataset_cls, model, embedding, loader, batch_si
     parser = argparse.ArgumentParser(description='PyTorch implementation of VDPWI')
     parser.add_argument('model_outfile', help='file to save final model')
     parser.add_argument('--dataset', help='dataset to use, one of [sick, msrvid, trecqa, wikiqa]', default='sick')
-    parser.add_argument('--word-vectors-dir', help='word vectors directory', default=os.path.join(os.pardir, os.pardir, 'Castor-data', 'embeddings', 'GloVe'))
+    parser.add_argument('--word-vectors-dir', help='word vectors directory', default=os.path.join(os.pardir, 'Castor-data', 'embeddings', 'GloVe'))
```

Comment:
Shouldn't there be two `os.pardir` items, since the first parent directory if the root Castor repo and the next higher up parent directory is castorini?

Reply:
https://github.com/castorini/Castor/blob/master/mp_cnn/__main__.py#L46

Then it is different from mpcnn. I think there should be not `castorini` directory.

Reply:
Oh sorry, my bad, your chance of one `os.pardir` is correct since we run the script from the repo root.